Test failure with rustc 1.33.0 / cargo 1.33.0

```
$ env CARGO_HOME=.cargo RUSTC_BOOTSTRAP=1 /usr/bin/cargo test -j8 -Z avoid-dev-deps --release --no-fail-fast
    Finished release [optimized] target(s) in 0.02s
     Running target/release/deps/fuzzy_matcher-32a736440ac848ef

running 4 tests
test clangd::tests::test_match_or_not ... ok
test skim::tests::test_match_or_not ... ok
test clangd::tests::test_match_quality ... ok
test skim::tests::test_match_quality ... ok

test result: ok. 4 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out

   Doc-tests fuzzy-matcher

running 2 tests
test src/skim.rs - skim::max (line 5) ... FAILED
test src/clangd.rs - clangd (line 5) ... FAILED

failures:

---- src/skim.rs - skim::max (line 5) stdout ----
error[E0432]: unresolved import `fuzzy_matcher::skim::fuzzy_matcher`
 --> src/skim.rs:6:40
  |
4 | use fuzzy_matcher::skim::{fuzzy_match, fuzzy_matcher};
  |                                        ^^^^^^^^^^^^^ no `fuzzy_matcher` in `skim`. Did you mean to use `fuzzy_match`?

error[E0425]: cannot find function `fuzzy_indices` in this scope
  --> src/skim.rs:12:24
   |
10 | let (score, indices) = fuzzy_indices("axbycz", "abc").unwrap();
   |                        ^^^^^^^^^^^^^ not found in this scope
help: possible candidates are found in other modules, you can import them into scope
   |
3  | use fuzzy_matcher::clangd::fuzzy_indices;
   |
3  | use fuzzy_matcher::skim::fuzzy_indices;
   |

error[E0659]: `fuzzy_matcher` is ambiguous (name vs any other name during import resolution)
 --> src/skim.rs:6:5
  |
4 | use fuzzy_matcher::skim::{fuzzy_match, fuzzy_matcher};
  |     ^^^^^^^^^^^^^ ambiguous name
  |
note: `fuzzy_matcher` could refer to the unresolved item imported here
 --> src/skim.rs:6:40
  |
4 | use fuzzy_matcher::skim::{fuzzy_match, fuzzy_matcher};
  |                                        ^^^^^^^^^^^^^
note: `fuzzy_matcher` could also refer to the extern crate imported here
 --> src/skim.rs:4:1
  |
2 | extern crate fuzzy_matcher;
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  = help: use `::fuzzy_matcher` to refer to this extern crate unambiguously
  = help: or use `crate::fuzzy_matcher` to refer to this extern crate unambiguously

thread 'src/skim.rs - skim::max (line 5)' panicked at 'couldn't compile the test', src/librustdoc/test.rs:321:13
note: Run with `RUST_BACKTRACE=1` environment variable to display a backtrace.

---- src/clangd.rs - clangd (line 5) stdout ----
error[E0432]: unresolved import `fuzzy_matcher::clangd::fuzzy_matcher`
 --> src/clangd.rs:6:42
  |
4 | use fuzzy_matcher::clangd::{fuzzy_match, fuzzy_matcher};
  |                                          ^^^^^^^^^^^^^ no `fuzzy_matcher` in `clangd`. Did you mean to use `fuzzy_match`?

error[E0425]: cannot find function `fuzzy_indices` in this scope
  --> src/clangd.rs:12:24
   |
10 | let (score, indices) = fuzzy_indices("axbycz", "abc").unwrap();
   |                        ^^^^^^^^^^^^^ not found in this scope
help: possible candidates are found in other modules, you can import them into scope
   |
3  | use fuzzy_matcher::clangd::fuzzy_indices;
   |
3  | use fuzzy_matcher::skim::fuzzy_indices;
   |

error[E0659]: `fuzzy_matcher` is ambiguous (name vs any other name during import resolution)
 --> src/clangd.rs:6:5
  |
4 | use fuzzy_matcher::clangd::{fuzzy_match, fuzzy_matcher};
  |     ^^^^^^^^^^^^^ ambiguous name
  |
note: `fuzzy_matcher` could refer to the unresolved item imported here
 --> src/clangd.rs:6:42
  |
4 | use fuzzy_matcher::clangd::{fuzzy_match, fuzzy_matcher};
  |                                          ^^^^^^^^^^^^^
note: `fuzzy_matcher` could also refer to the extern crate imported here
 --> src/clangd.rs:4:1
  |
2 | extern crate fuzzy_matcher;
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  = help: use `::fuzzy_matcher` to refer to this extern crate unambiguously
  = help: or use `crate::fuzzy_matcher` to refer to this extern crate unambiguously

thread 'src/clangd.rs - clangd (line 5)' panicked at 'couldn't compile the test', src/librustdoc/test.rs:321:13


failures:
    src/clangd.rs - clangd (line 5)
    src/skim.rs - skim::max (line 5)

test result: FAILED. 0 passed; 2 failed; 0 ignored; 0 measured; 0 filtered out

error: test failed, to rerun pass '--doc'
```

--- a/src/clangd.rs
+++ b/src/clangd.rs
@@ -3,7 +3,7 @@
 ///!
 ///! # Example:
 ///! ```edition2018
-///! use fuzzy_matcher::clangd::{fuzzy_match, fuzzy_matcher};
+///! use fuzzy_matcher::clangd::{fuzzy_match, fuzzy_indices};
 ///!
 ///! assert_eq!(None, fuzzy_match("abc", "abx"));
 ///! assert!(fuzzy_match("axbycz", "abc").is_some());

--- a/src/skim.rs
+++ b/src/skim.rs
@@ -3,7 +3,7 @@
 ///
 ///! # Example:
 ///! ```edition2018
-///! use fuzzy_matcher::skim::{fuzzy_match, fuzzy_matcher};
+///! use fuzzy_matcher::skim::{fuzzy_match, fuzzy_indices};
 ///!
 ///! assert_eq!(None, fuzzy_match("abc", "abx"));
 ///! assert!(fuzzy_match("axbycz", "abc").is_some());

